lib/vty.c

```diff
@@ -3075,9 +3075,20 @@ DEFUN (log_commands,
        log_commands_cmd,
        "log commands",
        "Logging control\n"
-       "Log all commands (can't be unset without restart)\n")
+       "Log all commands\n")
```

Comment:
The original use case of this was to disallow a use to "hide" changes by disabling logging.  I don't feel strongly about this, but think it worth explicitly discussing in the weekly meeting.